.travis.yml

```diff
@@ -3,9 +3,8 @@ sudo: false
 notifications:
   email: false
 node_js:
-  - "8" # latest stable
-  - "6" # LTS
-  - "4" # maintenance
+  - "10" # latest stable
+  - "8" # maintenance
```

Comment:
Why are this changes here? Are they needed for changing command output?

Reply:
Here Travis settings are transferred to the requirements of our repository.